src/Symfony/Bundle/DebugBundle/composer.json

```diff
@@ -18,13 +18,14 @@
     "require": {
         "php": ">=8.2",
         "ext-xml": "*",
+        "composer-runtime-api": ">=2.1",
```

Comment:
When installing the smallest possible Symfony app, we currently only require Symfony deps. So, that would be the first non-Symfony dependency that we require.

Reply:
I had the same thought until I realized the framework bundle already has this dependency:
https://github.com/symfony/symfony/blob/7.3/src/Symfony/Bundle/FrameworkBundle/composer.json